src/sensors/heading/gyroscope/GY50.cpp

```diff
@@ -11,7 +11,7 @@ namespace
 {
 const uint8_t kGyroAddress      = 105;
 const auto kMeasurementInterval = 100;
-const float kGyroSensitivity    = 0.07F;
+const float kGyroSensitivity    = 0.07f;
```

Comment:
:warning: **readability-uppercase-literal-suffix** :warning:
floating point literal has suffix 'f', which is not uppercase
```suggestion
const float kGyroSensitivity    = F```